src/Makevars.win

```diff
@@ -6,19 +6,10 @@
 # Copyright 2011         Uwe Ligges, Brian Ripley, and Josh Ulrich
 # Copyright 2018 - 2019  Jeroen Ooms
 
-RWINLIB=../windows/quantlib-1.16
+RWINLIB=../windows/quantlib-1.29
```

Comment:
That is awesome.  Any reason not to have taken last month's 1.30 release?   (1.29 should suit us for a while.)

Reply:
I copied from msys2 https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-quantlib/PKGBUILD